hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyArgs.java

```diff
@@ -54,6 +54,7 @@ public final class OmKeyArgs implements Auditable {
   private final boolean headOp;
   private final boolean forceUpdateContainerCacheFromSCM;
   private final Map<String, String> tags;
+  private final Integer partNumber;
```

Comment:
Do we need this since we already have `multipartUploadPartNumber`?

Reply:
Removed.